tests/fib/test_fib.py

```diff
@@ -498,6 +498,12 @@ def test_hash(add_default_route_to_dut, duthosts, tbinfo, setup_vlan,      # noq
     else:
         src_ip_range = SRC_IPV6_RANGE
         dst_ip_range = DST_IPV6_RANGE
+
+    if updated_tbinfo['topo']['name'] in ['t0-d18u8s4', 't0-isolated-d32u32s2'] and \
```

Comment:
we are missing many t0 isolated topos here, like full topo. if we only need to ignore this in t0 with service port, we can check against regex "t0-isolated-.*s\d+"